common/djangoapps/util/tests/test_db.py

```diff
@@ -161,3 +162,17 @@ def test_used_ids(self, times):
         for i in range(times):
             int_id = generate_int_id(minimum, maximum, used_ids)
             self.assertIn(int_id, list(set(range(minimum, maximum + 1)) - used_ids))
+
+
+class MigrationTests(TestCase):
+    """
+    Tests for lms/cms module itself.
+    """
+
+    def test_migrations_are_in_sync(self):
+        """
+        Tests that lms & cms migrations are in sync.
+        It will fail if migrations are not in sync.
```

Comment:
"Tests that the migration files are in sync with the models.

If this fails, you needs to run the Django command makemgirations."